api/src/main/java/org/pmiops/workbench/db/model/DbWorkspace.java

```diff
@@ -569,6 +570,15 @@ public void setBillingStatus(BillingStatus billingStatus) {
     this.billingStatus = DbStorageEnums.billingStatusToStorage(billingStatus);
   }
 
+  @Column(name = "billing_account_name")
+  public String getBillingAccountName() {
+    return billingAccountName;
+  }
+
+  public void setBillingAccountName(String billingAccountName) {
+    this.billingAccountName = billingAccountName;
+  }
+
```

Comment:
Re: the mapping between account names and project names, is that something guaranteed by the structure of one or the other name, or something we have to maintain with application code? Do all APIs that have access to change the name get checked against this invariant?